### ISSUE TITLE
Incorrect type and comment in tooltip for CommonJS export & assignment statement

### ISSUE THREAD

**Issue by pastelmind** [Microsoft/vscode#55877]
- VSCode Version: 1.25.1
- OS Version: Windows 10 Enterprise

Steps to Reproduce:

1. Create a JavaScript file with the following code:
    ```js
    /**
     * MyClass documentation
     */
    const MyClass = module.exports = class MyClass {
      //...
    }
    ```
2. When hovering the cursor over the first `MyClass`, the tooltip only shows `const MyClass: any`, and fails to display "MyClass documentation". When hovering the cursor over `module.exports` or the second `MyClass`, the tooltip does not appear at all.

Does this issue occur when all extensions are disabled?: Yes

Note: I also found a workaround, by adding an empty JSDoc comment before or after the actual JSDoc...

```js
/***/
/** MyClass documentation */
```

...or after it.

```js
/***/
/** MyClass documentation */
```

In both cases, the tooltip for the first `MyClass` displays:

>`const MyClass: typeof MyClass`
>MyClass documentation

However, the workaround is imperfect; the tooltip for `module.exports` and the second `MyClass` displays "MyClass documentation MyClass documentation".

**Comment by mjbvz** [comment]
This seems to be fixed in VS Code 1.26:

![screen shot 2018-08-24 at 12 48 44 pm](https://user-images.githubusercontent.com/12821956/44580912-1bec2e00-a79c-11e8-8c85-c8dcdb2b1dbc.png)

**Comment by isidorn** [comment]
Works nicely -> adding verified label
![screen shot 2018-08-30 at 15 59 19](https://user-images.githubusercontent.com/1926584/44856445-aede0a00-ac6d-11e8-8440-f6093c720d43.png)
